test/cpp/tensorexpr/test_simplify.cpp

```diff
@@ -635,17 +635,17 @@ TEST(Simplify, SimplifySub) {
   ASSERT_EQ(rhs->name_hint(), "x");
 }
 
-/// 2 * (1 - x) - 4 => -2 * (x + 3)
+/// 2 * (1 - x) - 4 => 2 * (-3 - x)
```

Comment:
Is there are reason to replace the previous tests? I get that there was a test coverage issue, but now we have the same issue just moved to be RHS dominant.

Reply:
Since we are only using positive integers to compute GCD, the factored out constant is also positive in all cases. So, I had to update the tests that were expecting negative factors. Those are the changes you are seeing here.

Reply:
Im suggesting we should have both versions of the tests.

Reply:
The version that factors out a negative integer, like `-2 * (x + 3)`, will never happen anymore, since the GCD is always positive. Are you referring to something else? Am I missing something here?